1.1-schnorr-signatures.ipynb

```diff
@@ -6,10 +6,10 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "import hashlib\n",
     "import random\n",
     "import util\n",
-    "from test_framework.key import generate_key_pair, ECKey, ECPubKey, jacobi_symbol, SECP256K1_FIELD_SIZE, SECP256K1_ORDER"
+    "from test_framework.key import generate_key_pair, ECKey, ECPubKey, jacobi_symbol, SECP256K1_FIELD_SIZE, SECP256K1_ORDER\n"
```

Comment:
Missing comma at the end of line causing error loading notebook

Reply:
fixed